README.md

```diff
@@ -15,15 +15,32 @@ To run the web server:
 npm run gulp
 ```
 
+If you need to run or debug the email server, you must supply the email
+password as an `env` var on the command line:
+
+```bash
+EMAIL_PASS="really bad password" npm run gulp
+```
+
+See the `app/configure` files for more `env` var overrides, such as `PORT`:
+
+```bash
+PORT=8080 EMAIL_PASS="really bad password" npm run gulp
+```
+
```

Comment:
So this is what we decided on? As an env variable? should we consider Firebase (https://firebase.google.com/) as an option?